### ISSUE TITLE
Implliment ability to send audio messages in chat.

### ISSUE THREAD

**Issue by SalemLockehart** [Aliucord/plugin-requests#1294]
### Plugin Description

I used the official app's feature to send audio messages with my partner quite a bit, and now that I've installed Aliucord, I've lost that ability/feature. It would be nice to have it back.

### Go into more detail...

Press a button to start recording an audio message, and send it directly into chat once it is finished recording.

### Request Agreement

- [X] I have made sure my plugin is possible and abides by the rules!

### ​

- [X] I did indeed check to make sure my plugin request is original!

**Comment by Juby210** [comment]
Already exists and plugin is called "VoiceMessages". https://github.com/mantikafasi/AliucordPlugins/blob/builds/VoiceMessages.zip

**Comment by SalemLockehart** [comment]
Thank you
